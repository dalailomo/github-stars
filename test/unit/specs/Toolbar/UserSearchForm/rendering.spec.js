import Vue from 'vue'
import Vuex from 'vuex'
import Vuetify from 'vuetify'
import UserSearchForm from '@/components/Toolbar/UserSearchForm'
import { getWrapper } from './../../../helpers'
import { emptyState, mocks } from './mocks'

Vue.use(Vuex)
Vue.use(Vuetify)

describe('User search form rendering', () => {
  const actions = {
    actionSetUsername: jest.fn(),
  }

  it('Renders form', () => {
    const wrapper = getWrapper(UserSearchForm, actions, emptyState, mocks, {})

    const template = wrapper.html()

    expect(template).toMatchSnapshot()
  })
})
