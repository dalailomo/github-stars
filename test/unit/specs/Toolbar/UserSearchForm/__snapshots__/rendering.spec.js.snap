// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User search form rendering Renders form 1`] = `
<form novalidate="novalidate" class="searchField">
  <div class="input-group input-group--text-field input-group--single-line primary--text">
    <label>Github user name</label>
    <div class="input-group__input">
      <input name="name" tabindex="0" aria-label="Github user name" type="text">
    </div>
    <div class="input-group__details">
      <!---->
    </div>
  </div>
  <button type="button" class="btn theme--dark green">
    <div class="btn__content">Search</div>
  </button>
</form>
`;
